lib/ansible_tower_client/base_model.rb

```diff
@@ -78,7 +91,7 @@ def key_to_attribute(key)
       end
 
       def generate_writer?
-        false
+        true
```

Comment:
I don't really like this idea.  This will generate attribute writer methods.  If those methods are publicly visible, users may get the wrong idea and expect the writer methods to update the API with the provided information, but it will just update the in memory AnsibleTowerClient object.